editoast/src/models/timetable.rs

```diff
@@ -97,13 +110,26 @@ impl Timetable {
                         .iter()
                         .filter(|stop| stop.duration > 0.)
                         .count() as i64;
+                    let rolling_stock = LightRollingStockModel::retrieve_conn(
+                        &mut conn,
+                        train_schedule.rolling_stock_id,
+                    )
+                    .unwrap();
+                    let rolling_stock_version = rolling_stock.unwrap().version;
+                    let train_validation = train_validation(
+                        train_schedule.clone().infra_version.unwrap(),
+                        train_schedule.clone().rollingstock_version.unwrap(),
+                        infra_version.clone(),
```

Comment:
```suggestion
                        &train_schedule.infra_version.unwrap(),
                        train_schedule.rollingstock_version.unwrap(),
                        &infra_version,
```

Reply:
this function does not need it's own copy of the data, it can take a reference just fine